### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by coolusaHD** [sinaatalay/RenderCV#26]
It would be nice if the duration and the text between two dates can be translated without creating his own template

**Comment by sinaatalay** [comment]
Thank you for opening an issue! This has been requested a couple of times, and an open pull request for this (#20) already exists, but I haven’t gotten the chance to review it yet. It requires big changes, so I need to think about it in my free time.

I will keep this issue open until this is implemented.

**Comment by AndreasKarasenko** [comment]
Copying from #14.
For people that want to change the language: you can directly edit the tex file created after the render cli command. You can then use the latex_to_pdf function from this library to recreate the pdf. Simple regex can automate that as well. I just changed all the date related things to German, the headers can be translated directly in the yaml.

E.g.
```
from rendercv.renderer import latex_to_pdf
import pathlib

path = pathlib.Path("./rendercv_output/test_CV.tex")
latex_to_pdf(path)
```
Change the path to the path of your file and enjoy pls.

**Comment by sinaatalay** [comment]
This issue should have been fixed with v1.9. Please comment if you encounter any problems.
